packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx

```diff
@@ -133,6 +137,22 @@ export const AccountsMenu = () => {
                 <GradientBefore isVisible={!isScrolledToTop} />
                 <Scroll ref={containerRef} onScroll={handleScroll}>
                     <AccountsList />
+                    {isDiscoveryButtonVisible && (
+                        <DiscoveryButtonContainer>
+                            <AccountsMenuNotice>
+                                <Translation id="TR_DISCOVERY_NEW_COINS_TEXT" isNested={false} />
+                            </AccountsMenuNotice>
+
+                            <Button
+                                variant="tertiary"
+                                size="tiny"
+                                icon="REFRESH"
+                                onClick={startDiscovery}
+                            >
+                                <Translation id="REFRESH" />
+                            </Button>
+                        </DiscoveryButtonContainer>
+                    )}
```

Comment:
consider abstracting this as well, as all the other parts have their own files already :)